packages/flutter/lib/src/painting/_network_image_web.dart

```diff
@@ -4,6 +4,8 @@
 
 
 import 'dart:async';
+import 'dart:html';
```

Comment:
Let's import this `as html`. This library has classes whose names clash with Flutter, e.g. `Element`.